scripts/apps/search/directives/ItemRepo.js

```diff
@@ -14,6 +14,16 @@ export function ItemRepo(
         link: function(scope, elem) {
 
             /*
+             * function to initialize default values on init or search provider change
+             */
+            scope.setDefaultValues = function() {
+                if (scope.repo && scope.repo.search && scope.repo.search.indexOf('scanpix') === 0) {
```

Comment:
yes, it was a quick workaround to have several Search Providers for the same instance with different login/password (this was not anticipated in the code).

I hope to remove all of this and put in plugins in the future, scanpix should not be hardcoded at all in the client, but this is the way it is done for the moment.